src/web/src/components/Icon/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { concatStyles } from "../../utils/concatStyles";
+import styles from "./Icon.module.scss";
+import BarChartSquare02 from "./commons/BarChartSquare02";
+import CheckCircle from "./commons/CheckCircle";
+import ChevronDown from "./commons/ChevronDown";
+import LogOut01 from "./commons/LogOut01";
+import SearchLg from "./commons/SearchLg";
+import Users01 from "./commons/Users01";
+import XClose from "./commons/XClose";
+
+const icons = {
+  ["bar-chart-square-02"]: BarChartSquare02,
+  ["users-01"]: Users01,
+  ["chevron-down"]: ChevronDown,
+  ["log-out-01"]: LogOut01,
+  ["search-lg"]: SearchLg,
+  ["check-circle"]: CheckCircle,
+  ["x-close"]: XClose,
+};
+
+const sizes = {
+  1: 16,
+  2: 20,
+  3: 24,
+};
```

Comment:
Adiciona um `TODO` só pra gnt n esquecer que isso tem q ser mudado em breve
```suggestion
// TODO: Get from theme when it's available
const sizes = {
  1: 16,
  2: 20,
  3: 24,
};
```